configure.ac

```diff
@@ -558,6 +558,7 @@ src/Makefile
 src/cli/Makefile
 src/ncp/Makefile
 src/core/Makefile
+src/diag/Makefile
```

Comment:
I'm confused what you are asking, because if I am understanding you right, isn't that behavior is already implemented via the `--enable-diag` configure flag?